accname/name/comp_name_from_content_menuitem.html

```diff
@@ -0,0 +1,107 @@
+
+      See more here:
+      https://github.com/w3c/accname/issues/205
+      https://github.com/w3c/accname/projects/1#card-42288231
+
+    */
+  </style>
+</head>
+<body>
+
+<!--TODO: Add test description-->
+<!--  TODO: Add tests for empty strings, empty alts, hidden elements (but not aria-hidden)-->
+  <h1>Named from Content for Anchor Menuitems</h1>
+  <ul role="menu">
+    <li>
+      <a role="menuitem" href="#" data-expectedlabel="Menuitem anchor named from content" data-testname="anchor name from content with
+         menuitem" class="ex">Menuitem anchor named from content</a>
+    </li>
+    <li>
+      <a role="menuitem" href="#" class="ex" data-expectedlabel="" data-testname="anchor name from content with
+         menuitem empty string"></a>
+    </li>
+  </ul>
+
+  <h2>Menuitem Anchor Author Overrides</h2>
+  <ul role="menu">
+    <li>
+      <a role="menuitem" href="#" aria-label="aria-label override" data-expectedlabel="aria-label override" data-testname="anchor name from content with menuitem aria-label override" class="ex">Description</a>
+    </li>
+    <li>
+      <a role="menuitem" href="#" aria-labelledby="aria-labelledby-image" data-expectedlabel="image" data-testname="anchor name from content with menuitem aria-labelledby override" class="ex">
+        <img src="#" id="aria-labelledby-image" alt="image">
+      </a>
+    </li>
+    <li>
+      <a role="menuitem" href="#" data-expectedlabel="image" data-testname="anchor name from content with menuitem nested img with alt override" class="ex">
+        <img src="#"  alt="image">
+      </a>
+    </li>
+  </ul>
+
+  <h1>Named from Content for Button Menuitems</h1>
+  <ul role="menu">
+    <li>
+      <button role="menuitem" data-expectedlabel="Menuitem button named from content" data-testname="button name from content with menuitem" class="ex">Menuitem button named from content</button>
+    </li>
+    <li>
+      <button role="menuitem" data-expectedlabel="" data-testname="button name from content with menuitem empty string" class="ex"></button>
+    </li>
+  </ul>
+
+  <h2>Menuitem Button Author Overrides</h2>
+  <ul role="menu">
+    <li>
+      <button role="menuitem" aria-label="Menuitem button named by aria label" data-expectedlabel="Menuitem button named by aria label" data-testname="button name from content with menuitem aria-label override" class="ex">Menuitem button named from content</button>
+    </li>
+    <li>
+      <button role="menuitem" aria-labelledby="button-span" data-expectedlabel="Button Span" data-testname="button name from content with menuitem empty string" class="ex"><span id="button-span">Button Span</span></span></button>
+    </li>
+    <li>
+      <button role="menuitem" data-expectedlabel="icon" data-testname="button name from content with menuitem with nested image alt" class="ex"><img src="#" alt="icon"></button>
+    </li>
+  </ul>
+
+  <h1>Named from Content for Table Heading Menuitems</h1>
```

Comment:
Is this a pattern you've seen elsewhere? I've seen menubuttons in table headers, but I don't recall seeing headers as menuitembuttons.